deploy/twingate-operator/templates/deployment.yaml

```diff
@@ -47,7 +47,7 @@ spec:
           - ./main.py
           - "-A"
           - "--standalone"
-          - "--verbose"
+          - "--{{- .Values.twingateOperator.logVerbosity | default "verbose" -}}"
```

Comment:
I believe this will work _except for_ if you want `INFO` log level, which the assumption in this code is that no argument will be supplied: https://github.com/zalando-incubator/kopf/blob/master/kopf/engines/logging.py#L142